site/docs/plugins/i18n.md

```diff
@@ -656,6 +656,9 @@ await i18n.loadLocalesDir("locales"); // async version
 i18n.loadLocalesDirSync("locales-2"); // sync version
 ```
 
+> Note that certain environments require you to use the `async` version.
+> For example, Deno Deploy does not support synchronous file operations.
```

Comment:
Should we add the reference link for other developers (https://deno.com/deploy/docs/runtime-fs)

Reply:
Can you perform any further changes in a subsequent PQ? As soon as something is ready for translation, we try not to alter the PQs anymore so that they can be translated more easily.

Reply:
PQ as **reason**?

I just thought that giving a reference will make other developers understand it easily.